tools/codec_enum_generator.py

```diff
@@ -0,0 +1,69 @@
+import subprocess
```

Comment:
A quick comment at the top saying how to run this tool, and a pointer to the file it generates, would be useful.

Imagine a year from now, someone coming in code, having to run this.

Reply:
I didn't know there would be a problem with & at javadoc . 😅 I modified it in codec files and generator.

And I added a description at the top of the generator.

Thank you.